src/api/app/assets/javascripts/webui/request.js

```diff
@@ -162,3 +165,17 @@ function loadDiffs(element){
     }
   });
 }
+
+function loadChanges(changesItem) {
```

Comment:
In this file, there is some JS code (like `loadDiffs` and probably more) which will be deprecated once the request show redesign is rolled out. Could we mark the code with TODOs? This will make the transition smoother.

Reply:
I added a couple of comments. Please have a look.